src/plugins/intel_gpu/thirdparty/CMakeLists.txt

```diff
@@ -49,6 +51,9 @@ if(ENABLE_ONEDNN_FOR_GPU)
                 "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
                 "-DDNNL_ENABLE_CONCURRENT_EXEC=ON"
                 "-DDNNL_ENABLE_PRIMITIVE_CACHE=OFF"
+                "-DDNNL_ENABLE_PRIMITIVE=${ONEDNN_ENABLED_PRIMITIVES}"
+                "-DDNNL_ENABLE_PRIMITIVE_GPU_ISA=${ONEDNN_ENABLED_ISA}"
+                "-DDNNL_ENABLE_WORKLOAD=INFERENCE"
```

Comment:
@dmitry-gorokhov Is it worrth to do the same inside CPU plugin while we have two copies of oneDNN?